### PR TITLE
Tickets/dm 1218

### PR DESCRIPTION
**Pull request by jdswinbank**

### CODE REVIEW

File: examples/runSingleFrameTask.py [hunk 1]
```diff
@@ -106,7 +107,7 @@ def run(display=False):
                 xy = s.getCentroid()
                 ds9.dot('+', *xy, ctype=ds9.CYAN if s.get("flags_negative") else ds9.GREEN, frame=frame)
                 ds9.dot(s.getShape(), *xy, ctype=ds9.RED, frame=frame)
-                ds9.dot('o', *xy, size=config.plugins["base_SincFlux"].radius2, ctype=ds9.YELLOW, frame=frame)
+                ds9.dot('o', *xy, size=config.plugins["base_CircularApertureFlux"].radii[0], ctype=ds9.YELLOW, frame=frame)
```
Comment (jdswinbank) [inline examples/runSingleFrameTask.py]: This line is too long per the coding standards.

File: examples/runSingleFrameTask.py [hunk 2]
```diff
@@ -72,9 +72,10 @@ def run(display=False):
     #
```
Comment (jdswinbank) [inline examples/runSingleFrameTask.py]: Should update the copyright statement on this file when editing it.

File: include/lsst/meas/base.h [hunk 3]
```diff
@@ -32,13 +32,11 @@
 #include "lsst/meas/base/InputUtilities.h"
 #include "lsst/meas/base/Algorithm.h"
 #include "lsst/meas/base/PsfFlux.h"
-#include "lsst/meas/base/SincFlux.h"
```
Comment (jdswinbank) [inline include/lsst/meas/base.h]: Should update the copyright statement on this file when editing it.

File: include/lsst/meas/base/ApertureFlux.h [hunk 4]
```diff
@@ -61,46 +61,34 @@ class ApertureFluxControl {
```
Comment (jdswinbank) [inline include/lsst/meas/base/ApertureFlux.h]: We can take the copyright thing as read, I guess!

File: examples/runSingleFrameTask.py (hunk 2 above)
Comment (TallJimbo) [inline examples/runSingleFrameTask.py]: Done.

File: examples/runSingleFrameTask.py (hunk 1 above)
Comment (TallJimbo) [inline examples/runSingleFrameTask.py]: Fixed.

File: include/lsst/meas/base/ApertureFlux.h (hunk 4 above)
Comment (TallJimbo) [inline include/lsst/meas/base/ApertureFlux.h]: Yup, think I've got them all now.

**Pull request merged**